source/commoncrypto/commoncrypto_ecc.c

```diff
@@ -0,0 +1,607 @@
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+#include <Security/SecKey.h>
+#include <Security/SecSignVerifyTransform.h>
+#include <Security/Security.h>
```

Comment:
trivial: our other projects subfolder by platform, not technology name, like:
`source/darwin/commoncrypto_ecc.c`?
`source/windows/bcrypt_ecc.c`?
instead of:
`source/bcrypt/bcrypt_ecc.c`
`source/commoncrypto/commoncrypto_ecc.c`